styles.css

```diff
@@ -132,6 +133,209 @@ body {
 }
 
 
+/* New code */
+    .grid-container {
+        display: grid;
+        grid-template-rows: repeat(4, auto);
+        grid-template-columns: 327px;
+        justify-content: center;
+        background-color: #E5E5E5;
+        padding: 114px 0;
+        row-gap: 88px;
+    }
+
+    .grid-item {
+        display: flex;
+        flex-direction: column;
+        border-radius: 16px;
+        border: 1px solid #dfe1e6;
+        background-color: #FFFFFF;
+    }
+
+    .grid-item img {
+        display: flex;
+        justify-content: center;
+        margin: 16px;
+    }
+
+    .grid-item h2 {
+        display: flex;
+        font-size: 32px;
+        line-height: 44px;
+        font-weight: 700;
+        margin: -3px 16px;
+    }
+
+    .details {
+        display: flex;
+        font-size: 13px;
+        line-height: 16px;
+        font-weight: 600;
+        margin: 0 16px;
+    }
+
+    .details ul {
+        margin: 0;
+        padding: 0;
+    }
+
+    .details li {
+        list-style-type: none;
+        display: flex;
+        align-items: center;
+        margin-top: 7px;
+    }
+
+    .details img {
+        margin: 16px 11px;
+    }
+
+    .worksText {
+        font-size: 15px;
+        line-height: 24px;
+        font-weight: 400;
+        margin: 8px 16px;
+    }
+
+    .worksTag {
+        font-size: 12px;
+        line-height: 16px;
+        font-weight: 500;
+    }
+
+    .worksTag ul {
+        margin: 2px 16px;
+        padding: 0;
+        display: flex;
+
+    }
+
+    .worksTag li {
+        list-style-type: none;
+        background-color:#EBEBFF;
+        padding: 4px 12px;
+        margin-right: 8px;
+        margin-top: 5px;
+        border-radius: 8px;
+        color: #6070FF;
+    }
+
+    .worksButton {
+        width: 124px;
+        height: 48px;
+        margin: 20px 16px;
+        font-size: 17px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #396DF2;
+        border: 1px solid #6070FF;
+        border-radius: 8px;
+        background-color: #FFFFFF;
+    }
+
+    .aboutMe {
+        padding: 114px 24px;
+    }
+
+    .aboutMe h1 {
+        font-size: 40px;
+        line-height: 52px;
+        font-weight: 700;
+        margin-top: 0;
+    }
+
+    .aboutPara {
+        margin-top: -7%;
+        font-size: 15px;
+        line-height: 24px;
+    }
```

Comment:
- [ ] Please follow the same Notion in the comment above about the sizes, colors, and spaces for this section font so it looks the exact same as shown in the Figma design.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/219881743-c142ffa1-c955-41ee-937e-5e1aee02430e.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219881765-8c86eb83-67ab-4981-a768-99faf10e8145.png)